tests/tfa_basic8.test

```diff
@@ -0,0 +1,375 @@
+<?php
+
+/**
+ * @file tfa_basic.test.
```

Comment:
I believe the Drupal 8 convention would be TfaBasic.php

You can remove the D7 code and comments - this will be in another branch